project/backend/api/views.py

```diff
@@ -350,6 +351,94 @@ def get_workspace_from_id(request):
                         'references':references,
                          'created_at':workspace.created_at,
                          }, status=200)
+def delete_entry(request):
+    id = int(request.GET.get("entry_id"))
+    entry = Entry.objects.filter(entry_id=id)
+    if entry.count() == 0:
+        return JsonResponse({'message': 'There is no entry with this id.'}, status=404)
+    entry.delete()
+def edit_entry(request):
+    id = int(request.GET.get("entry_id"))
+    entry = Entry.objects.filter(entry_id=id)
+    content = (request.GET.get("content"))
+    entry.content = content
+def delete_workspace(request):
+    id = int(request.GET.get("workspace_id"))
+    workspace = Workspace.objects.filter(workspace_id=id)
+    if workspace.count() == 0:
+        return JsonResponse({'message': 'There is no workspace with this id.'}, status=404)
+    workspace.delete()
+
+def delete_contributor(request):
+    id = int(request.GET.get("contributor_id"))
+    contributor = Contributor.objects.filter(contributor_id=id)
+    if contributor.count() == 0:
+        return JsonResponse({'message': 'There is no workspace with this id.'}, status=404)
+    contributor.delete()
```

Comment:
I believe this will delete the contributor entirely. Isn't this method supposed to remove the contributor from the workspace?

Reply:
Tried to resolve all the requested changes. Seems good to me now but I can change it further if anything is wrong.